### DISCUSSION TITLE
FLORIS model selection

### DISCUSSION THREAD

**Discussion by paulf81** [NatLabRockies/floris#179] [Q&A] [answered]
Hello everyone,

I am using FLORIS to make an estimation of the AEP produced by different wind farms, with different wind climates, turbines and layouts.
I get significantly different results ( % of AEP loss due to wake) with Jensen and Gauss_legacy velocity models.
I'd like to know which velocity deficit model do you recommend, without making any parameter adjustment, using the default parameters, to predict AEP losses due to wake?

I have read in https://www.nrel.gov/docs/fy19osti/72767.pdf that Gauss_legacy velocity deficit model provides better representation of wake characteristics. However, I have also read from some authors, from DONG Energy Wind Power, who prefer to use the Jensen model to predict AEP and power losses. Moreover, in reference [1], it is stated that Jensen model provides better estimation of AEP and wake losses for Horns Rev offshore wind farm.

Any recomendation or advice would be very helpful.

Many thanks in advance,

Nicolás Deza

[1] Recalibrating Wind Turbine Wake Model Parameters – Validating the Wake Model Performance for Large Offshore Wind Farms Thomas Sørensen, M.Sc, Per Nielsen, M.Sc. & Morten Lybech Thøgersen, M.Sc. EMD International A/S, Niels Jernes Vej 10, DK-9220 Aalborg East

**Comment by paulf81** [comment, marked as answer]
We recommend using the Gauss_legacy model.  The Jensen model has not been tuned and is just a standard out-of-the box implementation of the Jensen model.  There are other tunings that have happened for Jensen to give good performance in literature that are not included in FLORIS.  The Gaussian model has been tuned to SCADA data and LES data and we are more confident in that model.

**Reply by wenchaoyu** [reply]
Hello @paulf81,
If it is already published, woud it be possible to share the references about the tunning of the Gaussian model using SCADA data and LES data? Thanks.

**Reply by wenchaoyu** [reply]
Some discussions can be found in [https://github.com/NREL/floris/discussions/459](https://github.com/NREL/floris/discussions/459) where two references have been shared. Thanks @paulf81

**Reply by Nellytado** [reply]
Hi, this post was 2 years ago so I was wondering if the Jensen model has been tuned now?

**Reply by misi9170** [reply]
Hi @Nellytado , the Jensen model is included in FLORIS mostly for completeness and we do not use it regularly in analysis; it has therefore not been further tuned (and we do not currently have any plans to tune it).

**Comment by NicolasDD96** [comment]
Hello,

I read in "FLORIS: A Brief Tutorial" that some work is being done regarding Deep-Array effects, which seem to be quite relevant in large offshore wind farms to perform AEP calculations.
I would like to know, if possible, if you have an aproximate idea of when you will incorporate a Deep-Array model in FLORIS.

Thanks in advance

**Reply by paulf81** [reply]
Hi Nicolás, we have candidate models in place are are currently going through validation with an expectation for official release this summer.  If you'd like to be involved in early beta testing though please let us know!

**Reply by NicolasDD96** [reply]
Hi Paul, thank you for the clarification. I would like to participate in beta testing.
In fact, I have tried to couple a Deep Array model in FLORIS, but I gave up in the end, as I don't know the model very well. So, if I can participate in the beta testing, that would be great!
